tests/test_check_unknown.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from pydantic import BaseModel
+
+app_no_check = FastAPI(check_unknown=False)
+
+app_check = FastAPI(check_unknown=True)
+
+client_no_check = TestClient(app_no_check)
+client_ceck = TestClient(app_check)
```

Comment:
Not a big deal, but should probably be `client_check` and not `client_ceck` 👍

Reply:
yes you are right, changed that :)